src/context/Theme.tsx

```diff
@@ -415,6 +416,15 @@ export const alertTextPrimary = {
 };
 export const alertBackground = paperBackground;
 
+export const alertColorsReversed: {
+    [k in AlertColor]: { light: string; dark: string };
+} = {
+    success: { light: 'success.dark', dark: 'success.light' },
+    info: { light: 'info.dark', dark: 'info.light' },
+    warning: { light: 'warning.dark', dark: 'warning.light' },
+    error: { light: 'error.dark', dark: 'error.light' },
+};
```

Comment:
Eventually we need to review the alert colors. I think it is so weird showing a darker red on darker backgrounds. However, did not want to handle _all that_ in this PR.